toolchain/check/eval.cpp

```diff
@@ -769,6 +766,15 @@ static auto DiagnoseDivisionByZero(Context& context, SemIRLoc loc) -> void {
   context.emitter().Emit(loc, CompileTimeDivisionByZero);
 }
 
+// Get an integer at a suitable bit-width: either its actual width if it has a
+// fixed width, or the canonical width from the value store if not.
```

Comment:
I found it hard to work out how the parameters relate to the things referenced in this comment. How about something like this instead?

```suggestion
// Get an integer at a suitable bit-width: either `bit_width_id` if it is
// valid, or the canonical width from the value store if not.
```